packages/create-remix/__tests__/create-remix-test.ts

```diff
@@ -228,6 +228,25 @@ describe("create-remix CLI", () => {
     expect(fse.existsSync(path.join(projectDir, "app/root.tsx"))).toBeTruthy();
   });
 
+  it("works for GitHub username/repo/path combo (when dots exist in folder)", async () => {
+    let projectDir = getProjectDir("github-username-repo-path-dots");
+
+    let { status, stderr } = await execCreateRemix({
+      args: [
+        projectDir,
+        "--template",
+        "fake-remix-tester/nested-dir/folder.with.dots",
```

Comment:
I updated the `nested-dir-repo.tar.gz` file and copied the existing `stack/` directory to `folder.with-dots/` for this test